cs/dnc/src/grpc/project.json

```diff
@@ -44,7 +44,7 @@
     "frameworks": {
         "netstandard1.6": {
             "dependencies": {
-                "NETStandard.Library": "1.6.0"
+                "NETStandard.Library": "1.6.1"
```

Comment:
I think I saw more places with this dependency. Shouldn't we upgrade them all?

Reply:
No, that shouldn't be needed. Only Bond.Grpc needs this bumped dependency. The rest are fine with 1.6.0 until we get a bug report about behavior that's fixed in 1.6.1. The dependency solver should pick 1.6.1 in a project consuming Bond.Grpc, even if all the rest of the assemblies say they need 1.6.0.